docs/logs/customizing-the-sdk/customizing-the-sdk.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="$(MicrosoftExtensionsLoggingPkgVer)" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="$(MicrosoftExtensionsLoggingPkgVer)" /> <!--TODO: REMOVE WHEN FINISHED-->
```

Comment:
Is this needed in this PR?

Reply:
nope. this came from my larger rough draft and i'm stripping out stuff for the skeleton.
i'm using that for testing examples, didn't intend to include in in the PR, will remove now